properties.schema

```diff
@@ -80,6 +80,48 @@
                   "translatable": true,
                   "help": "Body text displayed in the popup"
                 },
+                "_imageAlignment": {
+                  "type": "string",
+                  "required": false,
+                  "default": "right",
+                  "inputType": {"type":"Select", "options":["left","top","right","bottom"]},
+                  "title": "Image alignment",
+                  "help": "Defines the alignment of the pop up image. Left: Image aligned to the left of the text area. Top: Image aligned above the text area. Right: Image aligned to the right of the text area. Bottom: Image aligned below the text area. The default alignment is `right`."
+                },
+                "_graphic": {
+                  "type": "object",
+                  "required": true,
```

Comment:
Not sure if this property is also now not required as we have set the `_src` property to not required?